doc/manuals/admin/yum.md

```diff
@@ -0,0 +1,47 @@
+## Using Yum repositories
+
+This document describes the guidelines of using FIWARE Yum repository to install Orion Context Broker. Provided configurations are corresponds to x86_64 architecture ($basearch) and CentOS/RHEL 7 OS ($releasever)
```

Comment:
the FIWARE yum repository

Reply:
s/are corresponds/correspond

Reply:
sorry for my ignorance but could you clarify to what $(basearch) and $(releasever) refer to?
They seem to be mispelled

Reply:
$basearch and $releasever are yum configuration variables, so $basearch=x86_64, $releasever=7, can you provide the way to clarify it?

Reply:
What about to include a link to yum documentation where both parameters are explained (if such documentation exists, of course)

Reply:
ok, will add.
https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/6/html/deployment_guide/sec-using_yum_variables

Reply:
fixed in c6e45d153045d7d51e08a4dc3c732985c22fe5c9